test/unit/src/issueLabelsHandlerSpec.js

```diff
@@ -1,111 +1,111 @@
-const issueLabelsHandler = require('../../../src/issueLabelsHandler');
+import test from 'ava';
+import sinon from 'sinon';
+import issueLabelsHandler from '../../../src/issueLabelsHandler';
 
-const expectDoNotCalculateOrAddPriorityLabel = (labels) => {
+const expectDoNotCalculateOrAddPriorityLabel = (t, labels) => {
```

Comment:
Instead of having a custom expect function you could also implement a [test macro](https://github.com/avajs/ava#test-macros).

Reply:
cool, haven't seen that feature before. Actually, i've just tried out and the  tests don't look that great with macro. It certainly a good use cases that they are showing in docs i.e. `test(macro, '2 + 2', 4);`, but when expecting some large objects and expectations are some stubs that need to be called. 
I will definitely keep this in mind for things that are passing in one line